plone/app/locales/locales/it/LC_MESSAGES/plone.po

```diff
@@ -6095,11 +6095,11 @@ msgstr "La cartella di destinazione ${target} non esiste."
 
 #: CMFPlone/controlpanel/browser/redirects.py:74
 msgid "Target path must not be a view."
-msgstr ""
+msgstr "L'url non deve essere una vista."
 
 #: CMFPlone/controlpanel/browser/redirects.py:69
 msgid "Target path must start with a slash."
-msgstr ""
+msgstr "L'url deve iniziare con \"/\"."
```

Comment:
In questo caso mi sembra che sia meglio usare "percorso"

Reply:
Concordo con @ale-rt